init/cli/logging.go

```diff
@@ -72,13 +82,28 @@ func (f *FileLoggers) SubProcessLogger(name string) launchlib.CreateLogger {
 }
 
 func (f *FileLoggers) OpenFile(path string) (*os.File, error) {
+	if _, ok := f.rotated[path]; !ok {
```

Comment:
Is `OpenFile` called more than once for a single path? I am just curious if you actually have to track what has been rotated and instead just always rotate on a call to `OpenFile`

Reply:
Yes. Without doing something like this, the startup file was being rotated multiple times per invocation of the launcher.